x-pack/plugins/infra/server/lib/alerting/inventory_metric_threshold/inventory_metric_threshold_executor.ts

```diff
@@ -200,6 +232,15 @@ const buildReasonWithVerboseMetricName = (
   return buildReason(resultWithVerboseMetricName);
 };
 
+const buildAlertData = (inventoryItem: any, nextState: any): AlertData => {
+  const threshold =
+    nextState === AlertStates.WARNING ? inventoryItem.warningThreshold! : inventoryItem.threshold;
```

Comment:
Using the correct types above would reveal that `inventoryItem.warningThreshold` and `inventoryItem.threshold` are actually of type `number[]`. This only accidentlly works because Elasticsearch flattens the array when indexing. How do we select the correct item? :thinking: We might need more help from the metrics team here.

Reply:
As I commented above this part is not needed anymore.